TealiumBranchExample/TealiumBranchExample/TealiumHelper.swift

```diff
@@ -30,7 +30,8 @@ class TealiumHelper {
         config.remoteAPIEnabled = true
         config.logLevel = .info
         config.collectors = [Collectors.Connectivity, Collectors.Lifecycle]
-        config.dispatchers = [Dispatchers.TagManagement, Dispatchers.RemoteCommands]
+        config.dispatchers = [//Dispatchers.TagManagement,
```

Comment:
Any reason we commented this out? If we don't want it we should probably just remove it instead of commenting.

Reply:
It's commented out because we removed the tagManagement from the dependency for remote commands. 
I left the comment there as a reminder for someone needing to test the webview remote command that he firstly needs to install tagmanagement before he can use it.